### DISCUSSION TITLE
What’s in _performance_evaluator.py?

### DISCUSSION THREAD

**Discussion by jbone** [online-ml/river#507] [Q&A]
There seem to be a number of (undocumented?) potentially useful metrics-reporting implementations in [the referenced file](https://github.com/online-ml/river/blob/master/river/metrics/_performance_evaluator.py).  Being a newb in this codebase I don’t understand the use of the _ prefix in various places;  are these deprecated, work-in-progress, etc.?

Thanks in advance!

**Comment by MaxHalford** [comment]
Actually that particular file is legacy code from the merger with scikit-multiflow. @smastelini @jacobmontiel can one of you confirm that we can remove it?

**Reply by smastelini** [reply]
Hey @MaxHalford, @jacobmontiel was involved in this portion of the merger. I think he's the best person to answer this question.
